### PR TITLE
fix(ci): bugfix for image_tag setting

### PR DESCRIPTION
**Pull request by Victoremepunto**
Only set the `IMAGE_TAG` variable in one place, this is to avoid a problem with frontend common build scripts 

testing with https://github.com/RedHatInsights/compliance-frontend/pull/2089

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -22,6 +22,41 @@ function job_cleanup() {
...
+    fi
+
+    echo "$image_tag"
+}
+
+is_change_request() {
+  [ -n "$ghprbPullId" ] || [ -n "$gitlabMergeRequestId" ]
+}
+
+get_change_request_image_tag() {
```
Comment (wcmitchell) [inline bootstrap.sh:45]: I don't see anywhere this gets used, and I also can't find the def for the above get_change_request_id, should these be the same?

File: bootstrap.sh [hunk 2]
```diff
@@ -22,6 +22,41 @@ function job_cleanup() {
...
+        image_tag="pr-$(get_change_request_id)-${git_short_hash}"
+    else
+        image_tag="$git_short_hash"
+    fi
+
+    echo "$image_tag"
+}
+
+is_change_request() {
+  [ -n "$ghprbPullId" ] || [ -n "$gitlabMergeRequestId" ]
```
Comment (maknop) [inline bootstrap.sh:42]: I thought one of those variables were always set so wouldn't this always resolve to `true`?

**Review by maknop — APPROVED** [review]
I left one comment but ultimately just for my knowledge. LGTM.

File: bootstrap.sh (hunk 1 above)
Comment (Victoremepunto) [reply bootstrap.sh]: thanks that was a horrible typo, naming is hard and I keep changing names :facepalm:  - well spotted @wcmitchell !!

File: bootstrap.sh (hunk 2 above)
Comment (Victoremepunto) [reply bootstrap.sh:42]: not if you run it outside of a "change request" context, like running the pr-check locally, which I did to test it :)

Comment (Victoremepunto) [reply bootstrap.sh:42]: also the bootstrap script runs not only in "pr_checks" - which may lead to pipeline runs that are not in a "change request context"

Comment (maknop) [reply bootstrap.sh:42]: Ohhh I see.

**Pull request merged**